src/shared/clients/ecsClient.ts

```diff
@@ -14,7 +14,7 @@ export type EcsResourceAndToken = {
     nextToken?: string
 }
 
-const MAX_RESULTS_PER_RESPONSE = 100
+const MAX_RESULTS_PER_RESPONSE = 10
```

Comment:
I would expect most/all services to have the same page size. Not per-service page sizes. 

But at least this fixes the issue in the short term.